packages/modelserver-node/src/client/model-server-client.ts

```diff
@@ -654,3 +656,13 @@ class DefaultTransactionContext implements TransactionContext {
 
 // A model update result indicating that the transaction was already closed or was rolled back
 const transactionClosed: ModelUpdateResult = { success: false };
+
+/**
+ * Query whether an `object` is any kind of `ModelServerCommand`.
+ *
+ * @param object an object
+ * @returns whether it is a `ModelServerCommand` of some kind
+ */
+export function isModelServerCommand(object: any): object is ModelServerCommand {
+    return ModelServerObject.is(object) && object.eClass.startsWith(ModelServerCommandPackage.NS_URI + '#//');
```

Comment:
Could we also expect a `type` attribute here (e.g. as in `ModelServerCommand`: `'type' in object`)?

Reply:
That would align this with the `ModelServerCommand.is` implementation, but actually I would argue that that function is overspecified. It is sufficient that the EClass identify the type of model object. If it's missing the `type` property, then it's invalid but it's still a `ModelServerCommand`.

Reply:
Alright sounds good, thanks for clearing that up!